client/channel/actions.go

```diff
@@ -274,9 +274,9 @@ func (c *Channel) subscribe(id uint32) chan agentpb.ServerResponsePayload {
 	return ch
 }
 
-func (c *Channel) publish(id uint32, resp agentpb.ServerResponsePayload) {
+func (c *ActionsChannel) publish(id uint32, resp agentpb.ServerResponsePayload) {
```

Comment:
Actually this is the channel for everything.

Reply:
Yep, I just don't find an appropriate name. Do you think it's better to leave it just `Channel`?

Reply:
Maybe I don't understand, but why did we create separate channel for jobs?

Reply:
Because the current channel is semi-synchronous, you can start the action and subscribe for the result, but with jobs it can be hundreds of messages.

https://github.com/percona/pmm-agent/blob/0fb1bd754a83108224c8901b8fe96fb267f8e9b9/client/channel/channel.go#L68

Reply:
I mean current channel implementation is not `just channel` it has some message handling logic that doesn't fit the jobs. Maybe I'm wrong, but I don't find a simple way to reuse the existing channel.

Reply:
we still can use the same mechanism that we use to collect qan metrics?

Reply:
What exact problem we have with the current implementation. Maybe I don't see something

Reply:
Refactored, now existing channel reused for jobs